Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -3590,6 +3590,22 @@ final class HTTPClientTests: XCTestCaseHTTPClientTestsBaseClass {
         XCTAssertEqual(.ok, response.status)
     }
 
+    func testClientWithMultipath() throws {
+        do {
+            var conf = HTTPClient.Configuration()
+            conf.enableMultipath = true
+            let client = HTTPClient(configuration: conf)
+            defer {
+                XCTAssertNoThrow(try client.shutdown().wait())
+            }
+            let response = try client.get(url: self.defaultHTTPBinURLPrefix + "get").wait()
+            XCTAssertEqual(.ok, response.status)
+        } catch let error as IOError where error.errnoCode == EINVAL {
```

Comment:
Just tried running this locally, it turns out `EPROTONOSUPPORT` is also sometimes thrown on Linux, so we need to add that to the list.

Reply:
Added this, I've checked the [documentation](https://www.mptcp.dev/implementation.html) and it seems that it could also raise an `ENOPROTOOPT` error, so I've added this one too to the list